test/unit_tests/test_common.py

```diff
@@ -16,12 +16,15 @@
 import unittest
 
 TEST_DIR = os.path.dirname(os.path.abspath(__file__))
+TEST_FILE = os.path.basename(os.path.abspath(__file__))
 SCRIPTS_DIR = os.path.abspath(os.path.join(TEST_DIR, os.pardir, os.pardir, "scripts"))
+SAMPLE_FILES_DIR = os.path.join(TEST_DIR, "sample_files")
 
 if not os.path.exists(SCRIPTS_DIR):
     raise ImportError("Cannot find scripts directory")
 
 sys.path.append(SCRIPTS_DIR)
+logging.disable(logging.CRITICAL)
```

Comment:
Isn't this the default behavior? I.e., equivalent to:
```suggestion
logging.disable()
```
If you think passing the value is important, please make it clear that `level` is an optional argument:
```suggestion
logging.disable(level=logging.CRITICAL)
```